demos/python_demos/human_pose_estimation_3d_demo/pose_extractor/CMakeLists.txt

```diff
@@ -18,8 +18,13 @@ add_library(${target_name} MODULE wrapper.cpp
                                   src/extract_poses.hpp src/extract_poses.cpp
                                   src/human_pose.hpp src/human_pose.cpp
                                   src/peak.hpp src/peak.cpp)
-target_include_directories(${target_name} PRIVATE src/ ${PYTHON_INCLUDE_DIRS} ${NUMPY_INCLUDE_DIR} ${OpenCV_INCLUDE_DIRS})
-target_link_libraries(${target_name} ${PYTHON_LIBRARIES} ${OpenCV_LIBS})
+target_include_directories(${target_name} PRIVATE
+    src "${Demos_SOURCE_DIR}/common"
+    ${PYTHON_INCLUDE_DIRS} ${NUMPY_INCLUDE_DIR} ${OpenCV_INCLUDE_DIRS})
+target_link_libraries(${target_name} PRIVATE
+    # we don't actually need Inference Engine, but the common headers depend on IE headers
+    ${InferenceEngine_LIBRARIES}
+    ${PYTHON_LIBRARIES} ${OpenCV_LIBS})
```

Comment:
Why did you increase dependencies for very insignificant changes? I dont see any sense here.

Reply:
What would you rather have me do? `arraySize` is in `common`; I don't want to duplicate it. I don't see how a dependency on `common` is a bad thing, given that every other demo relies on it too.

Reply:
What about `${InferenceEngine_LIBRARIES}`? I understand that 
> we don't actually need Inference Engine, but the common headers depend on IE headers

but I don't see any impact of `arraySize` and its advantages over `vector.size()`.

Reply:
> What about `${InferenceEngine_LIBRARIES}`?

The `pose_extractor` library doesn't actually become dependent on `inference_engine`, because it references no symbols from it. Although even if it did, I don't think it would be a problem, given that the demo (indirectly) depends on the IE library anyway.

> but I don't see any impact of `arraySize` and its advantages over `vector.size()`.

The advantage is that it works for arrays. :-) If you're asking why I want to use arrays here, it's because that allows us to avoid dynamic allocation and initialization each time the function is called.

Reply:
@eizamaliev Ping?

Reply:
As I've already said, I dont see any reasonable impact of it which compensate increased dependency.

Reply:
Which dependency do you object to?

Reply:
What wrong with `static const std::vector<std::pair<int, int> >`?
It can solve problem with 
>  dynamic allocation and initialization each time the function is called.

And will be the great compromise.

Reply:
A static array would still better than a static vector in this instance, because it would be initialized at compile time. And I don't see why I should apply a half-measure.

Reply:
OK, how about this? I just used `sizeof` to avoid pulling in the dependency.

Reply:
LGTM